pysmarthashtag/vehicle/trailer.py

```
@@ -0,0 +1,71 @@
+            retval["position_lamp_status"] = get_field_as_type(trailer_status, "trailerPosLampSts", int)
+
+            # Timestamp
+            if "updateTime" in vehicle_data["vehicleStatus"]:
+                retval["timestamp"] = datetime.fromtimestamp(
+                    int(vehicle_data["vehicleStatus"]["updateTime"]) / 1000
+                )
+
+        except KeyError as e:
+            _LOGGER.info(f"Trailer status info not available: {e}")
+        finally:
+            return retval
```

Ruff (0.15.2)

[warning] `return` inside `finally` blocks cause exceptions to be silenced

(B012)